### PR TITLE
@alloy => makefile improvements

### PR DESCRIPTION
**Pull request by orta**
Here's some special tea:
selfie-0

Now my branches are namespaced also.

`make pr` = push changes and create a PR in chrome
`make  push` = push new changes
`make pushf` = force push current

### CODE REVIEW

**Comment by alloy** [comment]
Mmmm, special T.

**Comment by orta** [comment]
was jelly of your branch names

File: Makefile [hunk 1]
```diff
@@ -121,10 +121,17 @@ beta: NOTIFY = 1
...
 
 pr: 
-	if [ "$(BRANCH)" == "master" ]; then echo "In master, not PRing"; else git push upstream "$(BRANCH)"; open -a "Google Chrome" "https://github.com/artsy/eigen/pull/new/artsy:master...$(BRANCH)"; fi
+	if [ "$(BRANCH)" == "master" ]; then echo "In master, not PRing"; else git push upstream "$(LOCAL_BRANCH):$(BRANCH)"; open -a "Google Chrome" "https://github.com/artsy/eigen/pull/new/artsy:master...$(BRANCH)"; fi
+
+push: 
+	if [ "$(BRANCH)" == "master" ]; then echo "In master, not pushing"; else git push upstream "$(LOCAL_BRANCH):$(BRANCH)";
+
+fpush: 
+	if [ "$(BRANCH)" == "master" ]; then echo "In master, not pushing"; else git push upstream "$(LOCAL_BRANCH)::$(BRANCH)" --force;
```
Comment (alloy) [inline Makefile]: Is the extra colon in `$(LOCAL_BRANCH)::$(BRANCH)` in error?

Comment (orta) [inline Makefile]: yes!

**Comment by orta** [comment]
fixed and ran all commands

**Comment by alloy** [comment]
Awesome! I forgot about this Makefile, I’m totes gonna start using it :+1:

**Pull request merged**